scripts/build-dtest-jars.sh

```diff
@@ -52,7 +52,7 @@ else
   if [[ "$CLEAN" == "true" ]]; then
     echo "Clean up $DTEST_JAR_DIR"
     rm -rf "$DTEST_JAR_DIR/cassandra-build"
-    rm "$DTEST_JAR_DIR/dtest*.jar"
+    rm -rf "$DTEST_JAR_DIR/dtest*.jar"
```

Comment:
Why? you should only be deleting the jar _files_ so what does `-r` do here?

Reply:
Because it failed to remove files. I got this error.
```
+ rm '.../cassandra-analytics/dependencies/dtest*.jar'
rm: .../cassandra-analytics/dependencies/dtest*.jar: No such file or directory
```

However, adding `-rf` is not the correct fix. It just hides the error. 

The correct `rm` command is `rm "$DTEST_JAR_DIR"/dtest*.jar`

Thanks for bringing it up!